in-app-wallet/typescript/src/in_app_wallet_examples.ts

```diff
@@ -0,0 +1,157 @@
+import { 
+    Connection, 
+    JsonRpcProvider, 
+    TransactionBlock,
+    SuiTransactionBlockResponse
+} from "@mysten/sui.js";
+import { rpcClient } from "typed-rpc";
+
+// The Key Service is Shinami's secure and stateless way to get access to the In-App Wallet
+const KEY_SERVICE_RPC_URL = "https://api.shinami.com/key/v1/<API_ACCESS_KEY>";
+
+// The Wallet Service is the endpoint to issue calls on behalf of the wallet.
+const WALLET_SERVICE_RPC_URL = "https://api.shinami.com/wallet/v1/<API_ACCESS_KEY>";
+
+// Shinami Sui Node endpoint + Mysten provided faucet endpoint:
+const connection = new Connection({
+    fullnode: 'https://node.shinami.com/api/v1/<API_ACCESS_KEY>',
+    faucet: 'https://faucet.testnet.sui.io/gas'
+});
+
+const suiProvider = new JsonRpcProvider(connection);
+const walletId = "<WALLET_ID>";
+const secret = "<SECRET>";
+const GAS_BUDGET = 5000000;
+
+// Key Service interaction setup
+interface KeyServiceRpc {
+    shinami_key_createSession(secret: string): string;
+}
+const keyService = rpcClient<KeyServiceRpc>(KEY_SERVICE_RPC_URL);
+  
+// Wallet Service interaction setup
+interface WalletServiceRpc {
```

Comment:
Might be good to separate these `typed-rpc` specs into individual files that people can simply download and use. Soon afterwards, we may even consider publishing them as a starting point of our TypeScript SDK.

Reply:
I think that can be put in an official typescript sdk helper lib that we release. For the sake of a tutorial, I'd rather leave it all in one file for ease of integration and readability from the user's perspective.